src/server/world.zig

```diff
@@ -948,6 +990,10 @@ pub const ServerWorld = struct { // MARK: ServerWorld
 				main.network.Protocols.genericUpdate.sendTimeAndBiome(user.conn, self);
 			}
 		}
+		if(self.lastTickTime + 50 < newTime) { // Tick very 50ms
```

Comment:
I think this should be done every server tick, otherwise we'll end up with an uneven load between server ticks. You can adjust the tick rate accordingly.